solidity/ts/testsuite/simulator/utils/utilities.ts

```diff
@@ -280,6 +295,7 @@ export const ensureShareTokenWrapperFactoryDeployed = async (client: WriteClient
 
 export const ensureAugurConstantProductMarketRouterDeployed = async (client: WriteClient) => {
 	const shareTokenWrapperFactoryAddress = await getShareTokenWrapperFactoryAddress()
+	await checkHookSalt();
```

Comment:
shouldn't this be in the vendoring script?

Reply:
Not sure! Can always move it if it works better there for some reason.